src/index.js

```diff
@@ -1,35 +1,48 @@
+// import _ from 'lodash';
+// import { add } from 'lodash';
 import './style.css';
+import Crud from './add&delete.js';
+// eslint-disable-next-line
```

Comment:
- [ ] [OPTIONAL] Kindly make sure you fix your linters instead of disabling them as this would help you learn how to write neat and DRY Javascript code. 🙏

Reply:
Sure, Thank you So much for noticing this out @Ugboaja-Uchechi